src/main/webapp/aboutus.html

```diff
@@ -41,17 +41,13 @@ <h2>Teammate B Name</h2>
       <li>Aspirational Hobby: </li>
       <li>Ask me About: </li>
     </ul>
-    <h2>Teammate C Name</h2>
-    <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
-    </ul>
-    <h2>Teammate D Name (delete if you team only has 3 students) </h2>
-    <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+    <link href="https://fonts.googleapis.com/css?family=Anton|Bungee+Shade&display=swap" rel="stylesheet">
```

Comment:
Stylesheet links should go up in the <head> tag with the other stylesheet. Convention is also to put the rel= field before the href= field so that the reader can quickly tell that something is a stylesheet.